src/containers/consumerOnboarding/components/productionAccessForm/Verification.tsx

```diff
@@ -46,6 +66,59 @@ const Verification: FC = () => {
       <div className="verification-divider vads-u-margin-top--4 vads-u-margin-bottom--1p5" />
       <SelectedAPIs selectedApis={apis} />
       <TermsOfServiceCheckbox termsOfServiceUrl={TERMS_OF_SERVICE_PATH} />
+      <VaModal
+        clickToClose
+        id="benefits-intake-attestation-modal"
+        large
+        modalTitle="Requirements for the Benefits Intake API"
+        onCloseEvent={(): void => setBenefitsIntakeModalVisible(false)}
+        visible={benefitsIntakeModalVisible}
+        primaryButtonText="Confirm"
+        onPrimaryButtonClick={(): void => setBenefitsIntakeModalVisible(false)}
+        secondaryButtonText="Cancel"
+        onSecondaryButtonClick={(): void => setBenefitsIntakeModalVisible(false)}
+        uswds
+      >
+        <p>
+          By accessing or using the{' '}
+          <span className="vads-u-font-weight--bold">Benefits Intake API</span> in the production
+          environment provided by VA, you must affirm and attest that the end user of your
+          application is:
+        </p>
+        <ol>
+          <li> A VA benefits claimant;</li>
```

Comment:
This extra space seems wrong.
```suggestion
          <li>A VA benefits claimant;</li>
```